### ISSUE TITLE
Spread syntax requires ...iterable[Symbol.iterator] to be a function

### ISSUE THREAD

**Issue by mkmik** [primatejs/primate-app#1]
```console
$ git clone
$ cd primate-app
$ npm i && npx -y primate

up to date, audited 174 packages in 1s

26 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
(node:35984) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
[info] primate 0.11.0
[info] running on localhost:6161
[error] Spread syntax requires ...iterable[Symbol.iterator] to be a function
```

The `[error] Spread syntax requires ...iterable[Symbol.iterator] to be a function` is repeated every time a page is loaded on `localhost:6161`:

```console
$ curl -i http://localhost:6161/
HTTP/1.1 500 Internal Server Error
Date: Tue, 14 Mar 2023 23:06:43 GMT
Connection: keep-alive
Keep-Alive: timeout=5
Transfer-Encoding: chunked
```

Arm64 macos. Tool versions:
```
$ node --version
v19.7.0
$ npm --version
9.6.1
$ npx --version
$ ls -l =node
lrwxr-xr-x 1 mkm admin 30 Mar 15 00:01 /opt/homebrew/bin/node -> ../Cellar/node/19.7.0/bin/node
$ ls -l =npm
lrwxr-xr-x 1 mkm admin 38 Mar 15 00:02 /opt/homebrew/bin/npm -> ../lib/node_modules/npm/bin/npm-cli.js
9.6.1
```

**Comment by terrablue** [comment]
Hey, thanks for reporting, this should be solved as of [ad95c7d](https://github.com/primatejs/primate-app/commit/ad95c7dd35418bb3698a085e4926fe1b4a66970b), could you please confirm?

@primate/redirect wasn't yet using the new handler syntax, see https://github.com/primatejs/primate-redirect/commit/38911017d40f18f960998e9147aabc02dddcc593

**Comment by mkmik** [comment]
it worked, thanks!
